src/rastervision/labels/object_detection_labels.py

```diff
@@ -19,15 +19,16 @@ def geojson_to_labels(geojson, crs_transformer, extent):
 
     for feature in features:
         # Convert polygon to pixel coords and then convert to bounding box.
-        polygon = feature['geometry']['coordinates'][0]
-        polygon = [crs_transformer.web_to_pixel(p) for p in polygon]
-        xmin, ymin = np.min(polygon, axis=0)
-        xmax, ymax = np.max(polygon, axis=0)
-        boxes.append(Box(ymin, xmin, ymax, xmax))
-
-        properties = feature['properties']
-        class_ids.append(properties['class_id'])
-        scores.append(properties.get('score', 1.0))
+        polygons = feature['geometry']['coordinates'][0]
```

Comment:
Seems like this wouldn't handle non-multipolygons. Perhaps we check the "type" field  here:

```python
geom_type = feature['geometry']['type']
if geom_type == 'polygon':
  ...
else if geom_type == 'multipolygon':
  ...
else:
  raise  Exception("Geometries of type {} are not supported in object detection labels.".format(geom_type))
```
or something along those lines.